source/Calamari.Tests/KubernetesFixtures/KubernetesContextScriptWrapperLiveFixtureAks.cs

```diff
@@ -134,8 +134,7 @@ public void UnreachableK8Cluster_ShouldExecuteTargetScript()
         [TestCase(true)]
         public void DiscoverKubernetesClusterWithAzureServicePrincipalAccount(bool setHealthCheckContainer)
         {
-            var serviceMessageCollectorLog = new ServiceMessageCollectorLog();
-            Log = serviceMessageCollectorLog;
+            Log = new InMemoryLog();
```

Comment:
I don't think we need this

Reply:
Yes, I think you are right. The `Log` is `InMemoryLog` by default: https://github.com/OctopusDeploy/Calamari/blob/d249e6bb18e38afc6249d15e32ea18ceb32ef797/source/Calamari.Tests/Helpers/CalamariFixture.cs#L25